src/js_command_match/constants.rs

```diff
@@ -0,0 +1,37 @@
+// Accounts and contracts
+pub const USE_ACCOUNT_ALIASES: [&str; 6] = [
+    "masterAccount",
+    "master-account",
+    "useAccount",
+    "use-account",
+    "accountId",
+    "account_id",
```

Comment:
I believe it should have hyphen here, instead of underscore:

```suggestion
    "account-id",
```

Otherwise, I get an error for a valid JS CLI command:

<img width="963" alt="image" src="https://github.com/near/near-cli-rs/assets/304265/33977780-1a51-46e4-80e5-a9159dc0180a">

<img width="961" alt="image" src="https://github.com/near/near-cli-rs/assets/304265/2a801322-3728-4272-8e36-fc609d24eeb7">

Reply:
underscrore's versions were removed from each of aliases